src/region/regionUtil.ts

```diff
@@ -1,5 +1,15 @@
 import * as React from 'react';
 import { Annotation, AnnotationRegion, Position } from '../@types';
+import { invertYCoordinate, Point, rotatePoint, translatePoint } from './transformUtils';
+
+const ROTATION_ONCE_DEG = -90;
```

Comment:
Should we add a comment explaining that these are the only values supported because they're the only ones that can be supplied by box-content-preview?